### DISCUSSION TITLE
Saucedacity Colors

### DISCUSSION THREAD

**Discussion by turtley12** [saucedacity/saucedacity#64] [General]
I am experimenting with some website ideas and I am trying to figure out what colors to use.

Does Saucedacity have existing colors?

If not, should Saucedacity have official colors?

Turtley12

**Comment by fitojb** [comment]
The current plan appears to be to take over the Tenacity brand, so I would hold off any efforts on this if I were you :)

**Reply by turtley12** [reply]
Oh ok. Thank you.

**Comment by generic-pers0n** [comment]
> I am experimenting with some website ideas and I am trying to figure out what colors to use.
> 
> Does Saucedacity have existing colors?
> 
> If not, should Saucedacity have official colors?
> 
> Turtley12

As said, we are looking at using Tenacity's name and branding in the future. However, if you would like to make a theme, we accept themes of all sorts! But yes, we're currently looking at using Tenacity's name and branding.
